python/lsst/meas/base/applyApCorr.py

```diff
@@ -244,6 +219,15 @@ def run(self, catalog, apCorrMap):
         # Wrap the task logger to a periodic logger.
         periodicLog = PeriodicLogger(self.log)
 
+        if isinstance(catalog, astropy.table.Table):
+            # Prepare the astropy table for use with this task.
+            self._prepAstropyTable(catalog)
+            xColumn = "x"
+            yColumn = "y"
```

Comment:
I would prefer if you make the column names configurable.  I know of at least one place I could use this on a pre-transformed table.